node/lib/package/LocalPackageSource.ts

```diff
@@ -90,47 +155,228 @@ export class LocalPackageSource extends PackageSource {
      * package, or null if the requested package is not found at the source
      */
     public async getPackageInfoAsync(name: string): Promise<PackageInfo | null> {
-        let packageJsonPath: string = path.join(this.cacheDirectory, name, "package.json");
+        if (name.startsWith("opent2t-schema-")) {
```

Comment:
Sure, I'll clean that up tonight. But you can go ahead and merge without waiting for this fix if you want.